index.html

```diff
@@ -232,74 +230,91 @@ <h4 id="title-contact">
-          </li>
-          <li class="language-item">
-            <img src="/assets/html.png" alt="html">
-            <h5>HTML</h5>
-          </li>
-          <li class="language-item">
-            <img src="/assets/css.png" alt="css">
-            <h5>CSS</h5>
-          </li>
-        </ul>
-        
-        <li class="skills-list-item">
-          <h3>Frameworks</h3>
-          <img src="/assets/chevron-right.png" alt="Frameworks">
-        </li>
-        <div class="skills-list-item-divider"></div>
-        <li class="skills-list-item">
-          <h3>Skills</h3>
-          <img src="/assets/chevron-right.png" alt="Skills">
-        </li>
-        <div class="skills-list-item-divider"></div>
-      </ul>
+            <!-- skills -->
+            <ul class="skills-list">
+                <li class="skills-list-item">
+                    <h3>Languages</h3>
+                    <img src="/assets/chevron-down.png" alt="Languages">
+                </li>
+                <ul class="languages">
+                    <li class="language-item">
+                        <img src="/assets/javascript.png" alt="JavaScript">
+                        <h5>JavaScript</h5>
+                    </li>
+                    <li class="language-item">
+                        <img src="/assets/html.png" alt="html">
+                        <h5>HTML</h5>
+                    </li>
+                    <li class="language-item">
+                        <img src="/assets/css.png" alt="css">
+                        <h5>CSS</h5>
+                    </li>
+                </ul>
 
-    </section>
-  </main>
+                <li class="skills-list-item">
+                    <h3>Frameworks</h3>
+                    <img src="/assets/chevron-right.png" alt="Frameworks">
+                </li>
+                <div class="skills-list-item-divider"></div>
+                <li class="skills-list-item">
+                    <h3>Skills</h3>
+                    <img src="/assets/chevron-right.png" alt="Skills">
+                </li>
+                <div class="skills-list-item-divider"></div>
+            </ul>
+        </section>
+
+        <!-- Contact me section -->
+        <section class="contact-me-section">
+            <div class="contact-me-section-container">
+                <h1>Contact me</h1>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                </p>
+                <form action="https://formspree.io/f/bbsimon" method="post" class="form-container">
+                    <input type="text" placeholder="Your name" minlength="3" maxlength="50" required>
+                    <input type="email" placeholder="Your email" required>
+                    <textarea name="message" id="message" cols="30" rows="10" minlength="5" maxlength="550" placeholder="Write your message here" required></textarea>
```

Comment:
- According to the project requirements, message input has a `maxlength="500"` attribute. You have used 550, please fix this 👍 👍